Public/Src/Cache/MemoizationStore/Vsts/BuildCacheServiceConfiguration.cs

```diff
@@ -108,6 +109,22 @@ public class BuildCacheServiceConfiguration
         /// </summary>
         public const bool DefaultOverrideUnixFileAccessMode = false;
 
+        /// <summary>
+        /// Default value indicating whether eager fingerprint incorporation is enabled.
+        /// </summary>
+        public const bool DefaultEnableEagerFingerprintIncorporation = false;
+
+        /// <nodoc />
+        public static readonly TimeSpan DefaultEagerFingerprintIncorporationNagleInterval = TimeSpan.FromMinutes(5);
+
+        /// <nodoc />
+        public const int DefaultEagerFingerprintIncorporationNagleBatchSize = 100;
+
+        /// <nodoc />
+        public static TimeSpan DefaultEagerFingerprintIncorporationExpiry = TimeSpan.FromDays(1);

```

Comment:
Technically these defaults violate the rule that, when shipped with no configuration, the new feature is off. For that to be true, they should both be set to infinity. The difference seems low risk, but I wouldn't want to explain myself in front of RCA if it goes wrong.